project/src/components/logo/logo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import PropTypes from "prop-types";
+
+function Logo({isFooterLogo}) {
+  return (
+    <div className="logo">
+      <Link to="/" className={isFooterLogo ? "logo__link logo__link--light" : "logo__link"}>
```

Comment:
```suggestion
      <Link to={AppRoute.MAIN} className={isFooterLogo ? "logo__link logo__link--light" : "logo__link"}>
```
Ты добавил енам AppRoute. Это круто! 
Давай использовать его по максимум